recipes/sbol_factory/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sbol_factory" %}
+{% set version = "1.0a12" %}
```

Comment:
This falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md
You can either request upstream for a stable release or release this under a dev label in conda-forge.

Reply:
Hi @ocefpaf , The developer released a stable version : https://github.com/SynBioDex/sbol_factory/releases/tag/v1.0a12
Thank you.

Reply:
`1.0a12` parses as alpha in pep440. Can you ask them to issue a tag/version that parses as stable?

Reply:
Hi @ocefpaf , ok done. A new version `v1.0` was released by the developer. I made the necessary changes for conda recipe. Thanks.